deployotron.actions.inc

```diff
@@ -977,9 +979,8 @@ namespace Deployotron\Actions {
      * {@inheritdoc}
      */
     public function validate() {
-      // This still prints output unless backend_options is set to FALSE.
-      if (!$this->drush('fra', array(), array('quiet' => TRUE, 'no' => TRUE))) {
-        return FALSE;
+      if (!$this->drush('fra', array(), array('quiet' => TRUE, 'no' => TRUE), FALSE)) {
+        return drush_set_error(dt("The drush command 'fra' could not be found."));
```

Comment:
Not too fond of this message. As I see it, drush fra might fail horrible for many reasons apart from not finding the command, and telling the user that the command is missing might cause a lot of confusion.

Could we capture the output somehow, and just discard it on error? Then we'd be able ta output it in case of error.